shared/bin/pacific/setup-ceph.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -e
+
+cd /ceph
+python3 -m pip install --upgrade pip
```

Comment:
I think this can be done in the general version of the script as it only updates the package manager, so it is not necessary to have a specific one for `Pacific`.